manifests/lge_d802.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+<remote name="enes" 
+            fetch="https://github.com/"/>
+<project path="device/lge/d802" name="LineageOS/android_device_lge_d802" remote="enes" revision="cm-14.1" />
```

Comment:
When you use "los" as remote you can drop "LineageOS/" from the name and drop "revision="cm-14.1"" as well.